modules/create-policy-categories.adoc

```diff
@@ -5,14 +5,38 @@
 [id="create-policy-categories_{context}"]
 = Creating policy categories
 
-You can create new policy categories from the system policies view.
+
+You can create new policy categories from the system policies view when creating or editing policies.
+
+
+{product-title-short} release 3.74 provides a new method to create and manage policy categories in {product-title-managed} or if you have the PostgreSQL database (Technical Preview) enabled. See the second procedure in the following section for more information. You can also configure policy categories using the `PolicyCategoryService` API object. For more information, navigate to *Help* -> *API reference* in the {product-title-short} portal.
+
+
+:FeatureName: PostgreSQL support
+include::snippets/technology-preview.adoc[]
+
 
 .Procedure
-. On the RHACS portal, navigate to *Platform Configuration* -> *Policies*.
+. On the {product-title-short} portal, navigate to *Platform Configuration* -> *Policy Management*.
 . From the *Policies* page, select the policy you want to edit.
 . Select *Actions* -> *Edit policy*.
 . Enter a new category name in the *Categories* field and then click *Create <category>*.
```

Comment:
This is no longer true. Now the user can only associate a policy category to the policy in the policy create/edit workflow.